js/views/storyeditview/storyeditview.js

```diff
@@ -261,15 +261,15 @@ var StoryEditView = Marionette.CompositeView.extend(
   },
 
   addPassageEndGameGroupSuccessNumberResult: function(name, left, top) {
-    this.addPassage(name, left, top, 'addPassageEndGameGroupSuccessNumberResult');
+    this.addPassage(name, left, top, 'passageEndGameGroupSuccessNumberResult');

```

Comment:
Yea I understand. Let's stick with what you've got. This is a simple find and replace refactor that we can do when we find we've locked down our models and can then simplify some of the naming.